### DISCUSSION TITLE
Can we make Replace Last Reply disabled unless there is text in the input field?

### DISCUSSION THREAD

**Discussion by Remowylliams** [oobabooga/text-generation-webui#3531] [Ideas]
Hi All,
  Dumb me I've wiped out 2 good section of story by accident just clicking on Replace Last Reply.
As far as I know, there's no recovering from this. If I'm wrong let me know.

The thing is, unless there is text in the input window (I'd like to see a min of 3 characters) I don't think there's a reason for it to function is there?

Hope I get some input on this. with so many buttons, it's easy to mis-click while your head is working on an idea.

**Comment by oobabooga** [comment]
Fixed here https://github.com/oobabooga/text-generation-webui/commit/8dbaa20ca8104aa5ead76dec13af3faa25f5d7e8

**Comment by Remowylliams** [comment]
I can confirm the fix works. Thank you so very much for patching it so quickly.
